src/popover/popover.ts

```diff
@@ -3,6 +3,7 @@ import {
   Directive,
   Input,
   Output,
+  HostListener,
```

Comment:
Isn't HostListener best practice, though? It's what they use in the Angular docs: https://angular.io/docs/ts/latest/guide/attribute-directives.html